bench_test.go

```diff
@@ -75,29 +90,28 @@ func BenchmarkSpan_1000Tags(b *testing.B) {
 	benchmarkWithOps(b, 0, 1000, 0)
 }
 
-func BenchmarkSpan_100BaggageItems(b *testing.B) {
-	benchmarkWithOps(b, 0, 0, 100)
-}
-
-func BenchmarkTrimmedSpan_100Events_100Tags_100BaggageItems(b *testing.B) {
-	var r CountingRecorder
-	t, err := NewTracer(
-		&r,
-		TrimUnsampledSpans(true),
-		WithSampler(neverSample),
-		TraceID128Bit(true),
-	)
-	if err != nil {
-		b.Fatalf("Unable to create Tracer: %+v", err)
-	}
-	benchmarkWithOpsAndCB(b, func() opentracing.Span {
-		sp := t.StartSpan("test")
-		return sp
-	}, 100, 100, 100)
-	if int(r) != b.N {
-		b.Fatalf("missing traces: expected %d, got %d", b.N, r)
-	}
-}
+//func BenchmarkSpan_100BaggageItems(b *testing.B) {
```

Comment:
intentional?

Reply:
yes, we don't support this yet and I would wait until extra fields take place in zipkin go.